docs/content/doc/installation/from-source.en-us.md

```diff
@@ -206,3 +206,86 @@ Similarly, a script for zsh-completion can be found at [`contrib/autocompletion/
 `.zshrc`.
 
 YMMV and these scripts may need further improvement.
+
+## Compile or cross-compile using Linux with Zig
+
+- Download: https://ziglang.org/download/
+- To install:
+```sh
+sudo rm -rf /usr/local/zig && tar -C /usr/local -xf zig-linux-x86_64-0.10.1.tar.xz
+```
+- Environment variable: `export PATH=$PATH:/usr/local/zig-linux-x86_64-0.10.1/`
+- Install cc
+```
+sudo apt install gcc
+which cc
+```
```

Comment:
Maybe
```suggestion
Follow [Getting Started of Zip](https://ziglang.org/learn/getting-started/#installing-zig) to install zig.
```
is enough?

I don't think it's appropriate to introduce too much about "how to install zip" into Gitea's documentation.